### ISSUE TITLE
You should make it clear that the program is Android only

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [TeamNewPipe/website#283]
https://newpipe.net/ does not make it very clear that the program runs only on Android (at least in the absence of some sort of emulator). Making this clear would aid some potential users.

**Comment by TiA4f8R** [comment]
I don't think so on the website itself: the website header of the home page cleary states that the app is made for Android devices only:

<img src="https://user-images.githubusercontent.com/74829229/171493042-c430c166-8822-4c77-8c2a-d694c274cdfb.png" width="300px" alt="newpipe_website_header_mobile">

This is also shown on higher screens (desktops, tablets) (it is also the case above download buttons (`Android 4.4+` for instance)), so in my opinion, we will not state this clearer on the website itself.

However, the title of the website is not including that the app is made only for Android, that we should modify because this is the element that search engines will show first about our website.

Anyway, thank you for the issue!
